ads/adssymbols.py

```diff
@@ -56,7 +56,8 @@ class Dummy(Array):
     _length_ = 1
     _type_ = c_ubyte
     def __init__(self, *args):
-        raise RuntimeError('Dummy ctypes object cannot be created or written')
+        pass # moved RuntimeError to where the variable is written
```

Comment:
This comment does not belong in the code, it is in the GIT history. This whole `__init__` can now be removed.